pages/dao-voting/index.vue

```diff
@@ -1,5 +1,7 @@
 <template>
```

Comment:
suggestion(non-blocking): this has nothing to do with you changes, but i noticed that the section with the indicator, countdown and tier-legend exists two times, could you create a ticket where we create a component for this?

Reply:
Yes that's a good idea. However, there are ideas to remove the second section entirely. So we need to discuss this for the next iteration of the 'leaderboard'.